src/ReportSystem/Concretes/BaseReport.cs

```diff
@@ -0,0 +1,16 @@
+using BorrowSystem;
+using Opcion1CanelitaTeam.ReportSystem.Interfaces;
+
+namespace Opcion1CanelitaTeam.ReportSystem.Concretes;
```

Comment:
I suggest moving to another directory called "Abstract", because there are concrete and interface directory so in order to have diferenced  the three types in the ReportSystem.

Reply:
Done.